src/DDTrace/Integrations/IntegrationsLoader.php

```diff
@@ -94,10 +94,8 @@ public function __construct(array $integrations)
                 '\DDTrace\Integrations\CakePHP\CakePHPSandboxedIntegration';
             $this->integrations[CodeIgniterSandboxedIntegration::NAME] =
                 '\DDTrace\Integrations\CodeIgniter\V2\CodeIgniterSandboxedIntegration';
-            if (\PHP_MAJOR_VERSION > 5) {
-                $this->integrations[CurlSandboxedIntegration::NAME] =
-                    '\DDTrace\Integrations\Curl\CurlSandboxedIntegration';
-            }
```

Comment:
Should be > 5.4 now, right?

Reply:
`ddtrace_config_sandbox_enabled()` always returns `false` for PHP 5.4.